src/ServicePulse.Host/vue/src/components/failedmessages/MessageList.vue

```diff
@@ -47,25 +49,18 @@ function labelClicked($event, index) {
       messages[x].selected = !messages[x].selected;
     }
 
-    clearSelection();
+    //clear selection
+    const selection = document.getSelection();
```

Comment:
This looks different to the old clearSelection because document.selection is not an official implementation (not recognised as being a valid property of document) and window.getSelection and document.getSelection are [equivalent](https://developer.mozilla.org/en-US/docs/Web/API/Document/getSelection), so only one needs to be checked. Likewise, empty() and removeAllRanges() are synonyms